### PR TITLE
RHBPMS-1698 -  Build & Deploy fails due to duplicate ProcessID after copying business process

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

File: jbpm-designer-backend/src/main/java/org/jbpm/designer/server/BusinessProcessCopyHelper.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                                      final Bpmn2TypeDefinition bpmn2ResourceType,
+                                      final CommentedOptionFactory commentedOptionFactory) {
+        this.ioService = ioService;
+        this.bpmn2ResourceType = bpmn2ResourceType;
+        this.commentedOptionFactory = commentedOptionFactory;
+    }
+
+    @Override
+    public boolean supports( final Path destination ) {
+        return ( bpmn2ResourceType.accept( destination ) || bpmn2ResourceType.accept( destination ) );
```
Comment (manstis) [inline jbpm-designer-backend/src/main/java/org/jbpm/designer/server/BusinessProcessCopyHelper.java]: Eh? Isn't this the same?

Comment (tsurdilo) [inline jbpm-designer-backend/src/main/java/org/jbpm/designer/server/BusinessProcessCopyHelper.java]: oops yes :) amending ..one sec

**Comment by manstis** [comment]
Other than the possible duplication in ```accept(..)``` this looks good to me.

@tsurdilo Glad you got it working!

**Comment by tsurdilo** [comment]
@manstis Thanks for your help on this (and I amended the duplication)!

File: jbpm-designer-backend/src/main/java/org/jbpm/designer/server/BusinessProcessCopyHelper.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
+/*
+* Copyright 2014 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline jbpm-designer-backend/src/main/java/org/jbpm/designer/server/BusinessProcessCopyHelper.java]: Some say it's best to "live for today" however some people like to reminice.

**Comment by manstis** [comment]
Minor issue with date in license; otherwise +1 to merge (when Jenkins is green)

**Comment by tsurdilo** [comment]
@manstis haha nice. I will update it too in org.drools.workbench.screens.guided.rule.backend.server.GuidedRuleEditorCopyHelper :)

**Comment by tsurdilo** [comment]
@manstis amended. thanks!

**Pull request merged**